go.mod

```diff
@@ -6,16 +6,16 @@ require (
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
 	github.com/go-logr/zapr v1.3.0
 	github.com/google/go-cmp v0.6.0
-	github.com/onsi/ginkgo/v2 v2.19.0
-	github.com/onsi/gomega v1.33.1
+	github.com/onsi/ginkgo/v2 v2.21.0
+	github.com/onsi/gomega v1.35.1
 	github.com/prometheus/client_golang v1.19.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stolostron/go-log-utils v0.1.2
 	github.com/stolostron/go-template-utils/v7 v7.0.1
 	github.com/stolostron/kubernetes-dependency-watches v0.10.0
-	k8s.io/api v0.31.1
-	k8s.io/apimachinery v0.31.1
-	k8s.io/client-go v0.31.0
+	k8s.io/api v0.32.9
```

Comment:
I think OCP v4.20 is Kubernetes v1.33? Might as well kick it there while we're at it?

Reply:
Nice - bringing in v0.33.3 caused `x/net` to update as well, so now it's just one commit!